docs/demos/mpi.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>I think you want the array creation itself to be outside the function you're timing. Also, maybe change "Single iteration time" to "Average wall-time per-iteration". Actually, even better, just print out the total wall-time, and keep the total number of iterations at 8 for all cases (I assume this is the number of processes you can launch...).</p><p> </p><p>Also, you can pass kwargs to the the function, so you don't have to redefine it below to take nthreads.</p><p> </p><p>Also, I'd separate the notebook into three small sections: (i) multithreading the FFT with pyfftw, (ii) multiprocessing with numpy as a backend, (iii) multiprocessing with pyfftw as a backend.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/powerbox/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>